rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -61,46 +61,77 @@ private BridgeSerializationUtils() {
         throw new IllegalAccessError("Utility class, do not instantiate it");
     }
 
-    public static byte[] serializeMap(SortedMap<Keccak256, BtcTransaction> map) {
-        int ntxs = map.size();
+    public static byte[] serializeRskTxWaitingForSignatures(Map.Entry<Keccak256, BtcTransaction> entry) {
+        byte[][] entryBytes = serializeRskTxWaitingForSignaturesSingleEntry(entry);
+        return RLP.encodeList(entryBytes);
+    }
 
+    public static byte[] serializeRskTxsWaitingForSignatures(SortedMap<Keccak256, BtcTransaction> map) {
+        int ntxs = map.size();
         byte[][] bytes = new byte[ntxs * 2][];
         int n = 0;
 
         for (Map.Entry<Keccak256, BtcTransaction> entry : map.entrySet()) {
-            bytes[n++] = RLP.encodeElement(entry.getKey().getBytes());
-            bytes[n++] = RLP.encodeElement(entry.getValue().bitcoinSerialize());
+            byte[][] entryBytes = serializeRskTxWaitingForSignaturesSingleEntry(entry);
+            bytes[n++] = entryBytes[0];
+            bytes[n++] = entryBytes[1];
         }
 
         return RLP.encodeList(bytes);
     }
 
-    public static SortedMap<Keccak256, BtcTransaction> deserializeMap(byte[] data, NetworkParameters networkParameters, boolean noInputsTxs) {
+    private static byte[][] serializeRskTxWaitingForSignaturesSingleEntry(Map.Entry<Keccak256, BtcTransaction> entry) {
```

Comment:
nit, but: wdyt about calling it `serializeRskTxWaitingForSignaturesEntry` ? without the `Single`

Reply:
Alright